modules/social_features/social_private_message/social_private_message.install

```diff
@@ -210,3 +210,31 @@ function social_private_message_update_8002() {
     $block->delete();
   }
 }
+
+/**
+ * Config import social_private_message default configs.
+ *
+ * The feature removal script did not succeed for social_private_message.
+ * Import the configs in an update hook.
+ */
+function social_private_message_update_8003() {
```

Comment:
We probably need to use hook_update_dependencies so it runs after social_core features removal